pages/information/hardware/usage.md

```diff
@@ -0,0 +1,89 @@
+
+Typically, PCF documentation breaks down the total carbon footprint of a product or device into life cycle stages: manufacturing, transportation, use, and end-of-life. Reported use or usage stage emissions estimates often represent lifetime emissions; annual contributions can be estimated based on the assumed device lifespan outlined in the PCF.
+
+Usage carbon varies between different devices, as illustrated by the following example:
+
+![usage-carbon](/assets/images/usage-carbon.png)
+
+*Source: Dell Product Carbon Footprint (PCF) documents for an example model of Laptop[^laptop], Desktop[^desktop], Monitor[^monitor], and Server[^server].*
+
+Servers can be associated with high usage emissions compared to devices like laptops or smartphones. This is largely due to higher power consumption associated with more powerful processors, large amounts of RAM, and multiple hard drives to store and process data, as well as more continuous usage patterns. Typically, server usage carbon emissions contribute the larger portion of their overall carbon footprint. In contrast, for user devices like laptops and desktops, embodied carbon forms a larger component than usage carbon. [Read more about embodied vs usage carbon emissions.](../lifecycle#embodied-vs-usage-carbon)
+
+{% include linkedHeading.html heading="Typical Energy Consumption (TEC)" level=3 %}
+
+When providing PCF data, manufacturers should include the assumptions and inputs used in the methodology. With regards to usage emissions, one such assumption of note is the [Typical Energy Consumption (TEC)](/glossary#typical-energy-consumption-tec) figure. TEC is an estimate of a device's expected annual energy consumption based on standard usage patterns. For devices like desktops, laptops, and smartphones, TEC is based on standard household usage. For servers, the estimation of TEC relies on several assumptions, including server utilisation, operational duration, and PUE of the data centre. 
+
+The TEC figure can be used in place of the estimated annual energy consumption figure approximated through the [high-level approach discussed earlier](#a-high-level-approach-for-estimating-usage-emissions). However, it's important to note that in some cases TEC may underestimate energy consumption compared to actual business usage patterns. This may stand for employee devices, where devices are typically used for longer durations (for example, 8 hours a day), or for servers operating at higher average utilisation levels than those assumed in the TEC calculations. Therefore, using TEC in subsequent estimations of emissions arising from business operations could result in lower emissions estimates than expected. 
+
+To ensure accurate and comparable emissions estimates across all hardware assets, it is recommended to clarify the assumptions made by manufacturers. Using consistent data sources and transparency over methodology is crucial.
+
+[See worked examples.](#worked-examples)
```

Comment:
The link to the Worked Examples is literally just below this link :D I don't think we need the link!?

Reply:
I was torn on this one because I thought the same, but there's a risk of things getting added later and the reader might lose where to see the examples

Reply:
removed it now